src/ConfigurationProcessor.php

```diff
@@ -57,26 +69,48 @@ public function processInstallation(array $configuration)
     {
         $this->io->write('<info>PrestaShop Module installer</info>');
 
-        if (!array_key_exists('modules', $configuration)) {
+        if (!isset($configuration['modules'])) {
             return;
         }
 
-        $nativeModules = $configuration['modules'];
-        $processManager = new ProcessManager(100, 8);
+        $modules = $configuration['modules'];
+        $updateFrequency = isset($configuration['update-frequency']) ?
+            $configuration['update-frequency'] :
+            self::DEFAULT_PROCESS_UPDATE_FREQUENCY
+        ;
+
+        $processes = isset($configuration['processes']) ?
+            $configuration['processes'] :
+            self::DEFAULT_PARALLEL_PROCESSES
+        ;
 
-        foreach ($nativeModules as $moduleName => $moduleVersion) {
-            $this->io->write(sprintf('<info>Looked into "%s" module (version %s)</info>', $moduleName, $moduleVersion));
+        $processManager = new ProcessManager($updateFrequency, $processes, $this->io);
+        $shouldOverWrite = $this->io->askConfirmation('Do you want to overwrite the existing modules? (y/n)');
```

Comment:
Oh I forgot this one, I think the default behaviour (with --no-interaction) should be NOT to overwrite
then you should have:
```
$shouldOverWrite = $this->io->askConfirmation('Do you want to overwrite the existing modules? (y/N)', false);
```

wdyt?

Reply:
it's a request of @eternoendless to have the overwrite by default, I have no strong opinion on the default value :angel:

Reply:
alright.. I think this will be bothersome if we need to answer this question each time we need to `composer install`
Personally I have a few scripts that allow me to install prestashop instance that are automated, if the overwrite is enabled by default (and thanks to my lousy connection) this will take a lot of time

I can understand why @eternoendless wants to enable the overwrite by default, but in this case I will renew my request about adding another parameter `--should-overwrite` or `--avoid-overwrite` to override this default behavior as explained here https://getcomposer.org/doc/articles/scripts.md#running-scripts-manually

this would allow to run `composer install --no-interaction -- --avoid-overwrite`

Reply:
> this would allow to run `composer install --no-interaction -- --avoid-overwrite`

it's not allowed by composer to pass extra parameters to install command: try it 👍

Reply:
Damn you're right, it interprets it as a package name..
The could you add this parameter in the extra config, this would at least allow to use a workaround by using: `composer config extra.prestashop.force-overwrite 0`

Reply:
I understand your point of view, but @eternoendless want this behavior by default :+1:

Reply:
A `composer install` will install the dependencies you expect to have. If you don't want to overwrite, you can use the `yes` command.